lua/zk/adapter.lua

```diff
@@ -1,12 +1,55 @@
 -- The interface between our plugin and zk
 
-local adapter = {}
+local M = {}
 
-function adapter.new()
+function M.new(title)
+  local cmd = "zk new"
+
+  if title ~= nil and title ~= "" then
+    cmd = string.format("zk new --title %s", title)
+  end
+
+  vim.fn.jobstart(
+    cmd,
+    {
+      on_stdout = function(j, d, e)
+        print(
+          string.format("[zk.nvim] on_stdout -> j: %s, d: %s, e: %s", vim.inspect(j), vim.inspect(d), vim.inspect(e))
+        )
+
+        vim.api.nvim_out_write("[zk.nvim] new zettel note created")
```

Comment:
To avoid confusion, maybe we could align the terminology between `zk` and `zk.nvim`? I usually don't mention anything Zettelkasten specific, only "note" and "notebook".
```suggestion
        vim.api.nvim_out_write("[zk.nvim] new note created")
```